pydig/resolver.py

```diff
@@ -60,7 +60,7 @@ def _args(self, domain, query_type):
         yield query_type.name
 
         # We only care about the result
-        yield '+short'
+        yield from ('+noall', '+answer') if query_type == query_type.AXFR else ('+short',)
```

Comment:
AXFR is not a DNS entry, it is a kind of DNS query aims to extract all the DNS entries of a DNS zone, for this reason the dig output is useless without the DNS entry type. So I decided to return the full entry and not the short version.

This chose is thought in order to maintain the retro compatibility with last version. In a second moment I suggest to redefine the output schema of the library.